<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-4SLAKQP" name="physicalRisk.ToUpdateSelfRiskAssessmentQuestion" tests="3" failures="1" timestamp="2023-05-10T09:46:13 IST" time="43.503" errors="0">
  <testcase name="addNewSelfRiskAssessmentQuestionInPhysicalRisk" time="37.601" classname="physicalRisk.ToUpdateSelfRiskAssessmentQuestion"/>
  <system-out>
    <![CDATA[Verified Question Label in Self Assessment page
Verified Save Button in Self Assessment page
The Captured data after creating question is :13 Testing
Clicked on Edit Button
Entered a new data into question textbox
Verified save button]]>
  </system-out>
  <testcase name="quit" time="1.772" classname="physicalRisk.ToUpdateSelfRiskAssessmentQuestion">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	User is not able to delete Self RiskAssessment in Physical Risk expected [Self RiskAssessment Deleted Successfully] but found [You cannot delete the existing Self RiskAssessment]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
User is not able to delete Self RiskAssessment in Physical Risk expected [Self RiskAssessment Deleted Successfully] but found [You cannot delete the existing Self RiskAssessment]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at Generic_Liberary.BaseClass.quit(BaseClass.java:278)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.afterRun(TestRunner.java:877)
at org.testng.TestRunner.run(TestRunner.java:599)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- quit -->
  <system-out/>
  <testcase name="deleteSelfRiskAssessment" time="4.130" classname="physicalRisk.ToUpdateSelfRiskAssessmentQuestion"/>
  <system-out/>
</testsuite> <!-- physicalRisk.ToUpdateSelfRiskAssessmentQuestion -->
